docs/Networking.asciidoc

```diff
@@ -214,17 +214,21 @@ for MM jobs:
 ```
 wget http://download.opensuse.org/tumbleweed/appliances/openSUSE-Tumbleweed-Minimal-VM.x86_64-Cloud.qcow2
 qemu-system-x86_64 -m 2048 -enable-kvm -vnc :42 -snapshot \
-  -netdev tap,id=qanet0,ifname=tap39,script=no,downscript=no -device virtio-net,netdev=qanet0,mac=00:00:00:00:00:01 \
+  -netdev tap,id=qanet0,ifname=tap40,script=no,downscript=no -device virtio-net,netdev=qanet0,mac=52:54:00:13:0b:4a \
   openSUSE-Tumbleweed-Minimal-VM.x86_64-Cloud.qcow2
 ```
 
-The image used here is of course just an example. Within the VM configure the network
-*like* this (you may need to adjust concrete IP addresses, subnets and interface names):
+The image used here is of course just an example. You need to make sure to assign a unique
+MAC address (e.g. by adjusting the last two figures in the example; this will not conflict with
+MAC addresses used by os-autoinst) and use a tap device not used at the same time by a SUT-VM.
+
+Within the VM configure the network *like* this (you may need to adjust concrete IP addresses,
+subnets and interface names):
```

Comment:
please wrap consistently at 80 characters per line. Your editor should already cover that :)

Reply:
I inferred our maximum for this document from what we use elsewhere in that document. I now wrapped everything (except code blocks) at 80 characters.